src/css.js

```diff
@@ -113,6 +113,18 @@ function getWidthOrHeight( elem, name, extra ) {
 		styles = getStyles( elem ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
+	// Support: IE11+
+	// Fix for edge case in IE 11. See https://github.com/jquery/jquery/issues/1764
+	if ( !support.reliableFullscreenBox && window.top !== window.self &&
+		document.msFullscreenElement ) {
+		// Support: IE<=11+
+		// Running getBoundingClientRect on a
+		// disconnected node in IE throws an error
+		if ( elem.getClientRects().length ) {
```

Comment:
We seem to be inconsistent now, e.g.: 0e4477c676db0427bb9b0bf39df8631501e62f24.

We have to use `jQuery.contains` on `compat`, not necessarily on `master`.

Note that here we don't really care if the node is disconnected but we care for `elem. getBoundingClientRect()` to not throw. These situations are equivalent, though.

This code is restricted to one browser, anyway so whatever works in IE11 is fine.